cosmosdb-checks/ttl-telematics-check-repo.js

```diff
@@ -0,0 +1,28 @@
+const { insertAlertIntoPg } = require("./pg-repo");
+var config = require("./config");
+const { environments } = require('../common/environments/index');
+async function performTTLTelematicsCheck(database, context)
+{
+    for(const env of environments) {
+        var querySpec = {
+            query: "SELECT count(1) as count FROM c where (GetCurrentTimeStamp()/1000 - c._ts > c.ttl+ 172800) and c._ts<(GetCurrentTimeStamp()/1000 - 604800)"
```

Comment:
we can use filter here (c.type="COMBINEDFEED" or c.type="TRIP" or c.type="EVENT")